src/torchmetrics/functional/image/ergas.py

```diff
@@ -62,7 +62,8 @@ def _ergas_compute(
             - ``'none'`` or ``None``: no reduction will be applied
 
     Example:
-        >>> preds = torch.rand([16, 1, 16, 16], generator=torch.manual_seed(42))
+        >>> gen = torch.manual_seed(42)
```

Comment:
would it be possible to set a global seed for all doctest so we do not need to have seed in our doctests?

Reply:
That would be tricky as the seed is set once and depends on what scope of test you use; this may expect that you always run the same sequence...

Reply:
okay dokay :]